modules/timeline.xql

```diff
@@ -15,25 +15,35 @@ response:set-header("Cache-Control", "no-cache"),
 
 let $start := request:get-parameter("start",1)
 let $count := request:get-parameter("count",10)
-let $timelineURLString := "https://exist-db.org/exist/apps/wiki/modules/feeds.xql?feed=eXist;lts&amp;start=" || $start || "&amp;count=" || $count
-let $data := httpclient:get(xs:anyURI($timelineURLString), false(), ())
-let $entries :=
-    for $entry in $data/httpclient:body//atom:entry
+let $timelineURLString := "https://exist-db.org/exist/apps/wiki/modules/feeds.xql?feed=eXist&amp;start=" || $start || "&amp;count=" || $count
+let $data := http:send-request(
+    <http:request method='get' />,$timelineURLString)
+  let $entries :=
+    for $entry in $data//atom:entry
     let $date := ($entry/atom:updated, $entry/atom:published)[1]
     order by xs:dateTime($date) descending
     return $entry
 
+return $entries
+
 for $entry in $entries
 let $dateStr := ($entry/atom:updated, $entry/atom:published)[1]
 let $date := xs:dateTime($dateStr/text())
 let $age := current-dateTime() - $date
 let $path := $local:WIKI_ROOT ||substring-after($entry/atom:link[@type eq 'blog']/@href, "/db/apps/wiki/data")
 let $title := $entry/atom:title/text()
-let $abstract := subsequence($entry/atom:content//xhtml:div/xhtml:p[1],1,1)
+let $abstract := subsequence($entry/atom:content/xhtml:div//xhtml:p[1],1,1)
 let $image := subsequence($entry/atom:content//xhtml:img[1]/@src,1,1)
 
-let $yeah := httpclient:head(xs:anyURI(concat($path,"/",$image)),false(),())
-let $checkresult := data($yeah/@statusCode) = 200
+let $yeah := 
```

Comment:
huh, that's an odd variable name (I see it was left unchanged). Maybe we can find a better one.

Reply:
Let `$statusOK`, perhaps.